src/portals/hope/pages/tools/ClaimStartVariants.tsx

```diff
@@ -169,10 +189,36 @@ const useClaimStartVariant = () => {
     )
   }
 
+  const bindEntrypoint = async (
+    entrypointId: string,
+    claimStartVariantId: string,
+  ) => {
+    return toast.promise(
+      bindEntrypointMutation({
+        variables: {
+          entrypointId,
+          claimStartVariantId,
+        },
```

Comment:
If we don't want to copy paste these for both `commonClaim` and `entrypoint` we could do something like this
```suggestion
  const bind = async (
    variables,
  ) => {
    return toast.promise(
      bindEntrypointMutation({
        variables,
```